dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -47,126 +43,69 @@
     private static final String OWNER = "owner";
     private static final String IDATE = "idate";
 
-    private ConvertToPOJOUtil(){
+    private ConvertToPOJOUtil() {
 
     }
 
     /**
      * Creates new instances of T given a List<Map> of DB results
+     * 
      * @param results
      * @param classToUse
      * @param <T>
      * @return
      * @throws Exception
      */
-    public static<T> List<T> convertDotConnectMapToPOJO(List<Map<String,String>> results,
-            final Class classToUse) throws ParseException, IllegalAccessException,
-            InvocationTargetException, InstantiationException, NoSuchMethodException, NoSuchFieldException {
+    public static <T> List<T> convertDotConnectMapToPOJO(List<Map<String, Object>> results, final Class classToUse)
+            throws ParseException, IllegalAccessException, InvocationTargetException, InstantiationException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalAccessException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.reflect.InvocationTargetException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.InstantiationException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)